src/main/java/net/mcreator/ui/modgui/ItemGUI.java

```diff
@@ -73,7 +73,7 @@ public class ItemGUI extends ModElementGUI<Item> {
 
 	private StringListProcedureSelector specialInformation;
 
-	private final JSpinner stackSize = new JSpinner(new SpinnerNumberModel(64, 1, 64, 1));
+	private final JSpinner stackSize = new JSpinner(new SpinnerNumberModel(64, 1, 99, 1));

```

Comment:
Most slots are limited to 64 afaik, so even if the stack size here is set to 99, in practice, this will be 64 in most slots, which may confuse the users

Reply:
I tested with vanilla containers and there doesn't seem to be a limit on slots